java/ql/test/query-tests/security/CWE-489/DebuggableAttributeEnabledTest.ql

```diff
@@ -2,20 +2,19 @@ import java
 import semmle.code.xml.AndroidManifest
 import TestUtilities.InlineExpectationsTest
 
-class DebuggableAttributeTrueTest extends InlineExpectationsTest {
-  DebuggableAttributeTrueTest() { this = "DebuggableAttributeEnabledTest" }
+class DebuggableAttributeEnabledTest extends InlineExpectationsTest {
+  DebuggableAttributeEnabledTest() { this = "DebuggableAttributeEnabledTest" }
 
   override string getARelevantTag() { result = "hasDebuggableAttributeEnabled" }
 
   override predicate hasActualResult(Location location, string element, string tag, string value) {
     tag = "hasDebuggableAttributeEnabled" and
-    exists(AndroidXmlAttribute androidXmlAttr |
-      androidXmlAttr.getName() = "debuggable" and
-      androidXmlAttr.getValue() = "true" and
-      not androidXmlAttr.getLocation().getFile().getRelativePath().matches("%build%")
+    exists(AndroidApplicationXmlElement androidAppElem |
+      androidAppElem.isDebuggable() and
+      not androidAppElem.getFile().isInBuildDirectory()
```

Comment:
Same as above: 

```suggestion
      not androidAppElem.getFile().(AndroidManifestXmlFile).isInBuildDirectory()
```

Reply:
Updated in commit [bee310d](https://github.com/github/codeql/pull/9939/commits/bee310d9e0c0e8aa9927572d28dad4bd5c414981).